src/components/HeaderWithCloseButton.js

```diff
@@ -76,7 +77,10 @@ const HeaderWithCloseButton = props => (
                     </TouchableOpacity>
                 </Tooltip>
             )}
-            <Header title={props.title} />
+            <View style={[styles.flex1, styles.flexColumn]}>
+                <Header title={props.title} />
+                {props.stepCounter && <Text style={[styles.mutedTextLabel, styles.mt1]}>{props.translate('stepCounter', props.stepCounter)}</Text> }
```

Comment:
Could I see it without the `mt1` - I think if it is a bit more compact, it might feel a little nicer.

Reply:
Sure, this is without the `mt1`:
<img width="750" alt="Screen Shot 2021-09-14 at 15 04 10" src="https://user-images.githubusercontent.com/6829422/133340238-afc1fb3c-10fa-4d8d-8f71-7ebbbbd7068f.png">
<img width="750" alt="Screen Shot 2021-09-14 at 17 04 06" src="https://user-images.githubusercontent.com/6829422/133340241-5e349065-639b-4b82-b093-562a0c4670c2.png">